gulpfile.js

```diff
@@ -577,101 +578,102 @@ export const makeZip = gulp.series(release, async function () {
-      "Apps/**/.eslintrc.json",
-      "Apps/Sandcastle/.jshintrc",
-      "!Apps/Sandcastle/gallery/development/**",
-      "packages/engine/index.js",
-      "packages/engine/index.d.ts",
-      "packages/engine/LICENSE.md",
-      "packages/engine/README.md",
-      "packages/engine/Source/**",
-      "!packages/engine/.gitignore",
-      "packages/widgets/index.js",
-      "packages/widgets/index.d.ts",
-      "packages/widgets/LICENSE.md",
-      "packages/widgets/README.md",
-      "packages/widgets/Source/**",
-      "!packages/widgets/.gitignore",
-      "Source/**",
-      "Source/**/.eslintrc.json",
-      "Specs/**",
-      "!Specs/e2e/*-snapshots/**",
-      "Specs/**/.eslintrc.json",
-      "ThirdParty/**",
-      "favicon.ico",
-      ".eslintignore",
-      ".eslintrc.json",
-      ".prettierignore",
-      "scripts/**",
-      "gulpfile.js",
-      "server.js",
-      "index.cjs",
-      "LICENSE.md",
-      "CHANGES.md",
-      "README.md",
-      "web.config",
-    ],
-    {
-      base: ".",
-    }
-  );
-
-  const indexSrc = gulp
+  const src = gulp
     .src("index.release.html")
-    .pipe(gulpRename("index.html"));
-
-  return streamToPromise(
-    mergeStream(
-      packageJsonSrc,
-      enginePackageJsonSrc,
-      widgetsPackageJsonSrc,
-      builtSrc,
-      staticSrc,
-      indexSrc
+    .pipe(gulpRename("index.html"))
+    .pipe(packageJsonSrc)
+    .pipe(enginePackageJsonSrc)
+    .pipe(widgetsPackageJsonSrc)
+    .pipe(
+      gulp.src(
+        [
+          "Build/Cesium/**",
+          "Build/CesiumUnminified/**",
+          "Build/Documentation/**",
+          "Build/Specs/**",
+          "Build/package.json",
+          "packages/engine/Build/**",
+          "packages/widgets/Build/**",
+          "!Build/Specs/e2e/**",
+          "!Build/InlineWorkers.js",
+          "!packages/engine/Build/Specs/**",
+          "!packages/widgets/Build/Specs/**",
+          "!packages/engine/Build/minifyShaders.state",
+          "!packages/engine/Build/package.noprepare.json",
+          "!packages/widgets/Build/package.noprepare.json",
+        ],
+        {
+          encoding: false,
+          base: ".",
+        }
+      )
     )
-      .pipe(
-        gulpTap(function (file) {
-          // Work around an issue with gulp-zip where archives generated on Windows do
-          // not properly have their directory executable mode set.
-          // see https://github.com/sindresorhus/gulp-zip/issues/64#issuecomment-205324031
-          if (file.isDirectory()) {
-            file.stat.mode = parseInt("40777", 8);
-          }
-        })
+    .pipe(
+      gulp.src(
+        [
+          "Apps/**",
+          "Apps/**/.eslintrc.json",
+          "Apps/Sandcastle/.jshintrc",
+
+          "packages/engine/index.js",
+          "packages/engine/index.d.ts",
+          "packages/engine/LICENSE.md",
+          "packages/engine/README.md",
+          "packages/engine/Source/**",
+          //"!packages/engine/.gitignore",
```

Comment:
Are these necessary or helpful to keep around?

Reply:
Nope, these can be removed.